### DISCUSSION TITLE
v0.14.2-beta1

### DISCUSSION THREAD

**Discussion by vasqued2** [vasqued2/ha-teamtracker-card#97] [Announcements]
- Fix rendering issues on iPads and other legacy devices

<hr /><em>This discussion was created from the release <a href='https://github.com/vasqued2/ha-teamtracker-card/releases/tag/v0.14.2-beta1'>v0.14.2-beta1</a>.</em>

**Comment by vasqued2** [comment]
HA release 2024.8.0 updated HA's legacy device list to include many new devices that had previously been considered modern.  There was also a defect that caused all iPads to be classified as legacy devices.  The teamtracker_card has as incompatibility with the code for legacy devices, which causes the card to sporadically render incorrectly.  Before the 2024.8.0 release, this was not detected because most devices used the modern code.

HA is going to fix the defect which is misclassifying iPads as legacy devices.  This will fix the rendering problem for iPads.  This will not address the incompatibility with other legacy devices, however.  

Addressing the incompatibility with other legacy devices required a significant rewrite of the teamtracker-card.  It is possible that something that functionality that currently works on modern devices broke with the changes that were made.  

Because the problem occurs sporadically and is dependent upon the sensor state, the device, and the the order that multiple cards appear on a dashboard tab, testing has is extremely complicated.  I have performed an initial round of testing and have released this beta version.  However I am going to hold off on the full release for some time to give time for feedback from those using the card on multiple devices, both modern and legacy.

If you have problems with the beta release.  Please provide the following information when reporting a problem:
- Description of the problem
- HA version
- Device experiencing the problem
- Browser or app used and version
- Version of the card
- Number of cards on the dashboard, placement of the card on the dashboard (1st, 2nd, 3rd, etc.), and the state of the sensor displayed on the card
- Ideally a screenshot of the sensor attributes from the Develop tools page
- If you have the browser or app User Agent, please provide that as well

Thanks for your help to ensure this release is solid before it is fully released.

**Comment by PRProd** [comment]
Nice work @vasqued2!  👍🏻 

I can confirm that the rendering issue I detailed in #95 is corrected in this beta.  v0.14.1 still showed the issue, I updated to v0.14.2-beta1 and the card rendered as expected.
